### PR TITLE
Changes case on paths for Component folder.

### PR DESCRIPTION
**Pull request by andersekbom**

### CODE REVIEW

**Comment by andersekbom** [comment]
Double bummer: I made a typo in the branch name. ;)

**Comment by roelineburger** [comment]
Oh no, what a silly mistake! Thank you for pointing this out and noticing.

File: youmoni/app/devicedetails.tsx [hunk 1]
```diff
@@ -7,12 +7,12 @@ import {
   Platform,
   ScrollView,
 } from "react-native";
-import CustomInput from "../src/components/Input";
+import CustomInput from "../src/Components/Input";
```
Comment (roelineburger) [inline youmoni/app/devicedetails.tsx:10]: 😨 Thank you for picking this up!

Comment (andersekbom) [reply youmoni/app/devicedetails.tsx:10]: Was only an issue when running on Linux, so no worries. ;)

**Pull request merged**